.tests/testScript_units.m

```diff
@@ -32,7 +32,7 @@
 timeNeeded = scd(timeNeeded,'month')
 
 %% str2u examples
-str2u('kg-m�/s^3')
+str2u('kg-m²/s^3')
```

Comment:
This and following places are caused by UTF-8 encoding. Please refer to the test results to ensure it works correctly